src/diffusers/models/attention.py

```diff
@@ -339,6 +389,88 @@ def forward(
         return hidden_states
 
 
+@maybe_allow_in_graph
+class SkipFFTransformerBlock(nn.Module):
+    def __init__(
+        self,
+        dim: int,
+        num_attention_heads: int,
+        attention_head_dim: int,
+        kv_input_dim: int,
+        kv_input_dim_proj_use_bias: bool,
```

Comment:
```suggestion
        kv_input_dim: int,
        kv_input_dim_proj_use_bias: bool,
```

Is this really needed for MUSE? Can we delete it?

Reply:
We actually do need the `kv_input_dim` for muse. We can get rid of the `...use_bias` arg and hardcode along with the other bias's though. Would that be ok?

Reply:
Yeah `kv_input_dim` is needed here, so let's keep it.